src/olympia/accounts/serializers.py

```diff
@@ -78,19 +78,28 @@ class UserProfileSerializer(PublicUserProfileSerializer):
     picture_upload = serializers.ImageField(use_url=True, write_only=True)
     permissions = serializers.SerializerMethodField()
     fxa_edit_email_url = serializers.SerializerMethodField()
+    reviewer_name = serializers.CharField(
+        min_length=2, max_length=50,
+        validators=[OneOrMorePrintableCharacterAPIValidator()])
 
     class Meta(PublicUserProfileSerializer.Meta):
         fields = PublicUserProfileSerializer.Meta.fields + (
-            'display_name', 'email', 'deleted', 'last_login', 'picture_upload',
-            'last_login_ip', 'read_dev_agreement', 'permissions',
-            'fxa_edit_email_url', 'username',
+            'deleted', 'display_name', 'email', 'fxa_edit_email_url',
+            'last_login', 'last_login_ip', 'permissions', 'picture_upload',
+            'read_dev_agreement', 'reviewer_name', 'username'
         )
         writeable_fields = (
             'biography', 'display_name', 'homepage', 'location', 'occupation',
-            'picture_upload',
+            'picture_upload', 'reviewer_name',
```

Comment:
If we're exposing `reviewer_name` in the serializer response it should be documented in the api docs.  

How is displaying/editing `reviewer_name` going to work in the frontend?  Is there an issue logged to expect it and show it to the user?  There won't be any problem with having a writable field on the form that is only sometimes available?

Reply:
I'll add docs and file an issue for that yeah - it was mentioned in the tasks to do in the addons-server issue.

If the user is a reviewer, they are supposed to see an extra field. I don't think it should be too much of a problem on the frontend.

Reply:
https://github.com/mozilla/addons-frontend/issues/7806